server/src/test/java/io/spine/examples/shareaware/server/e2e/given/E2ETestUser.java

```diff
@@ -0,0 +1,159 @@
+        this.walletId = GivenWallet.walletId(userId);
+        CreateWallet createWallet = createWallet(walletId);
+        client.onBehalfOf(userId)
+              .command(createWallet)
+              .postAndForget();
+    }
+
+    /**
+     * Allows user to take a look at all the {@code EntityState}s with the provided type.
+     */
+    public <S extends EntityState> ImmutableList<S> lookAt(Class<S> type) {
+        return client
+                .onBehalfOf(userId)
+                .select(type)
+                .run();
+    }
+
+    /**
+     * Allows user to send the provided command to the server.
+     */
+    public void command(CommandMessage command) {
+        client.onBehalfOf(userId)
+              .command(command)
+              .postAndForget();
+    }
+
+    /**
+     * Subscribes the user to receive the event of the passed type.
+     *
+     * <p>Returns only the {@link CompletableFuture} that stores the event
+     * without {@link Subscription}.
+     *
+     * @see E2ETestUser#subscribeToEvent(Class)
+     */
+    protected <E extends EventMessage> CompletableFuture<E>
+    subscribeToEventAndForget(Class<E> type) {
+        SubscriptionOutcome<E> subscriptionOutcome = subscribeToEvent(type);
+        return subscriptionOutcome.future();
+    }
+
+    /**
+     * Subscribes the user on changes of the passed type of the {@code EntityState}.
+     *
+     * <p>Returns only the {@link CompletableFuture} that stores the {@code EntityState}
+     * without {@link Subscription}.
+     *
+     * @see E2ETestUser#subscribeToState(Class)
+     */
+    protected <S extends EntityState> CompletableFuture<S>
+    subscribeToStateAndForget(Class<S> type) {
+        SubscriptionOutcome<S> subscriptionOutcome = subscribeToState(type);
+        return subscriptionOutcome.future();
+    }
+
+    /**
+     * Subscribes the user to receive the event of the passed type.
+     */
+    protected <S extends EventMessage> SubscriptionOutcome<S> subscribeToEvent(Class<S> type) {
+        CompletableFuture<S> future = new CompletableFuture<>();
+        Subscription subscription = client
+                .onBehalfOf(userId)
+                .subscribeToEvent(type)
+                .observe(future::complete)
+                .post();
+        return new SubscriptionOutcome<>(future, subscription);
+    }
+
+    /**
+     * Subscribes the user on changes of the passed type of the {@code EntityState}.
+     */
+    protected <S extends EntityState> SubscriptionOutcome<S> subscribeToState(Class<S> type) {
+        CompletableFuture<S> future = new CompletableFuture<>();
+        Subscription subscription = client
+                .onBehalfOf(userId)
+                .subscribeTo(type)
+                .observe(future::complete)
+                .post();
+        return new SubscriptionOutcome<>(future, subscription);
+    }
+
+    /**
+     * Cancels the passed subscription.
+     *
+     * @see io.spine.client.Subscriptions#cancel(Subscription)
```

Comment:
In general, we don't want to point readers to some external API unless we really require them to go there.

Let's remove this one.